frontend/src/views/producer/producer-modal/index.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import Modal, { Content } from '../../../components/modal';
+import Typography from '../../../components/typography';
+import { connect } from '../../../utils';
+import { withTheme } from 'styled-components';
+import Icon from 'antd/lib/icon';
+
+export default withTheme(
+  connect('store')(
+    class AppModal extends Component {
```

Comment:
Should this component be called ProducerModal instead of AppModal?

Reply:
Yes.
it should be producer modal